packages/frontend/pages/MessageBoard.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useRef, useEffect} from 'react'
+import { v4 as uuidv4 } from 'uuid'
+
+export function Message( {message} ) {
+    return (
+        <div>
+            <h4>Name: {message.name}</h4>
+            <p>Message:</p>
+            <p>{message.msg}</p>
+            <p>[DEBUG]MessageID: {message.id}</p>
+            <br></br>
+        </div>
+    )
+}
+
+export function MessageList({ messages }) {
+    return (
+        messages.map(message => {
```

Comment:
This can be made inline: `messages.map(message => <Message key=.../>`)

Reply:
neat, fixed